packages/artifact/src/internal/utils.ts

```diff
@@ -42,11 +66,48 @@ export function isRetryableStatusCode(statusCode?: number): boolean {
   const retryableStatusCodes = [
     HttpCodes.BadGateway,
     HttpCodes.ServiceUnavailable,
-    HttpCodes.GatewayTimeout
+    HttpCodes.GatewayTimeout,
+    429 // TODO, change when a new version of @actions/http-client gets released
   ]
   return retryableStatusCodes.includes(statusCode)
 }
 
+export function isThrottledStatusCode(statusCode?: number): boolean {
+  if (!statusCode) {
+    return false
+  }
+
+  // TODO, change when a new version of @actions/http-client gets released
+  return statusCode === 429
```

Comment:
Why is this new version not released yet? Would be good to consume those changes since it's already merged to master

Reply:
New version is not yet released yet. I'm following up on that: https://www.npmjs.com/package/@actions/http-client

Reply:
Version 1.0.7 is out